src/toil/wdl/utils.py

```diff
@@ -32,3 +77,79 @@ def dict_from_JSON(JSON_file):
         else:
             json_dict[d] = data[d]
     return json_dict
+
+
+def write_mappings(parser: AnalyzeWDL) -> None:
+    """
+    Intended to take an AnalyzeWDL instance and prints the final task dict,
+    and workflow dict.
+
+    Does not work by default with toil since Toil actively suppresses stdout
```

Comment:
We shouldn't merge a thing that says it "does not work". Maybe make the filename a parameter or make it take a stream, and document it as writing to that filename or stream?

Reply:
This one is my fault.  It's an old function I used for debugging and developing the WDL parser.  It prints the intermediate data structures so that you can look at what it's doing before creating the final file.  It's useful to leave in, but it needs the `--debugWorker` flag if I remember right to print stdout normally.

Reply:
It looks like this function no longer prints stuff to stdout, so perhaps the docstring is just outdated. 

I'll make the filename a parameter and update the docstring. I think this function is useful to keep - I use it a lot for debugging as well.